deploy.ts

```diff
@@ -0,0 +1,5 @@
+export const deploy = async () => {
+  console.log("Deploy of Schemas Starting...");
+
+  console.log("TODO: Actual Deploy");
```

Comment:
Is there an issue for this already?

Reply:
Now there is: https://github.com/LibertyDSNP/schemas/issues/3